build.sbt

```diff
@@ -715,6 +716,30 @@ lazy val kernelDefaults = (project in file("kernel/kernel-defaults"))
     unidocSourceFilePatterns += SourceFilePattern("io/delta/kernel/"),
   ).configureUnidoc(docTitle = "Delta Kernel Defaults")
 
+
+lazy val sparkKernelDsv2 = (project in file("spark-kernel-dsv2"))
+  .dependsOn(kernelApi)
+  .dependsOn(kernelDefaults)
+  .dependsOn(spark % "test->test")
+  .settings(
+    name := "delta-spark-dsv2",
```

Comment:
Can you please remind me the future release plan here? Separate JAR? Merge this module into the `delta-spark` module eventually?